src/locales/ko/battle.ts

```diff
@@ -62,4 +62,9 @@ export const battle: SimpleTranslationEntries = {
   "drainMessage": "{{pokemonName}}[[로]]부터\n체력을 흡수했다!",
   "regainHealth": "{{pokemonName}}[[는]]\n기력을 회복했다!",
   "fainted": "{{pokemonNameWithAffix}}[[는]] 쓰러졌다!",
+  "retryBattle": "Would you like to retry from the start of the battle?",
+  "receivedNewModifier": "You received\n{{modifierName}}!",
+  "unlockedSomething": "{{unlockedThing}}\nhas been unlocked.",
+  "congratulations": "Congratulations!",
+  "beatModeFirstTime": "{{speciesName}} beat {{gameMode}} Mode for the first time!\nYou received {{newModifier}}!",
```

Comment:
```suggestion
  "retryBattle": "이 배틀의 처음부터 재도전하시겠습니까?",
  "receivedNewModifier": "{{modifierName}}[[를]]\n손에 넣었다!",
  "unlockedSomething": "{{unlockedThing}}[[가]]\n해금되었다.",
  "congratulations": "축하합니다!",
  "beatModeFirstTime": "{{speciesName}}[[가]] {{gameMode}} 모드를 처음으로 클리어했다!\n{{newModifier}}[[를]] 손에 넣었다!",
```
Translations for Korean locales